### ISSUE TITLE
Bug: CreateMany Returns Error Complaining "Argument `skipDuplicates` Does Not Exist On Enclosing Type"

### ISSUE THREAD

**Issue by Congyuwang** [Brendonovich/prisma-client-rust#173]
Calling create_many() returns this error:
```
Error executing query: P2009 - Failed to validate the query: `Argument does not exist on enclosing type.` at `Mutation.createManyLocation.skipDuplicates`
```

function called: `client.location().create_many(many_data).exec().await?`

prisma-client-rust = { git = "https://github.com/Brendonovich/prisma-client-rust", tag = "0.6.1" }

Here is a minimal example Repo to reproduce the problem: https://github.com/Congyuwang/rust-prisma-create-many-debug

The model in the example looks like this:
```prisma
generator client {
  provider = "cargo prisma"
  output   = "../src/prisma.rs"
}

datasource db {
  provider = "mongodb"
  url      = env("DB_URL")
}

model Location {
  id          String @id @default(auto()) @map("_id") @db.ObjectId
  description String
}
```

The main function looks like this:
```rust
pub mod prisma;

#[tokio::main]
async fn main() {
    dotenv::dotenv().unwrap();
    let client = prisma::new_client().await.unwrap();
    client.location().create_many(vec![
        ("a".to_string(), vec![]),
        ("b".to_string(), vec![]),
        ("c".to_string(), vec![]),
    ]).exec().await.unwrap();
}
```

To reproduce, clone the repo https://github.com/Congyuwang/rust-prisma-create-many-debug, add a mongodb `DB_URL` and `cargo run --release`.

**Comment by Brendonovich** [comment]
Hmm, forgot that skipDuplicates isn't available everywhere. Will need to make that a database-specific feature.

**Comment by Congyuwang** [comment]
Right. According to https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#createmany: `skipDuplicates` is only supported by databases that support [ON CONFLICT DO NOTHING](https://www.postgresql.org/docs/9.5/sql-insert.html#SQL-ON-CONFLICT).

**Comment by Congyuwang** [comment]
skipDuplicates availability:
- Postgres: YES (https://www.postgresql.org/docs/9.5/sql-insert.html#SQL-ON-CONFLICT)
- mysql: YES (https://www.mysqltutorial.org/mysql-insert-ignore/)
- mssql: NO (https://stackoverflow.com/questions/53611915/alternatives-for-insert-ignore-into-for-sql-server)
- sqlite: YES (https://www.sqlite.org/lang_insert.html)
- mongoDB: DOUBTFUL / NO (https://stackoverflow.com/questions/34153058/mongodb-multi-document-insert-ignore-custom-duplicate-field-error)

**Comment by Brendonovich** [comment]
The availability of `skip_duplicates` is defined by the `CreateSkipDuplicates` capability, which is [hardcoded](https://github.com/Brendonovich/prisma-engines/blob/1a9235ce5c50bb629cb63235ca8b5ca82198cada/libs/datamodel/connectors/sql-datamodel-connector/src/postgres_datamodel_connector.rs#L91) on each connector. The generator and runtime will need to check whether the capability is available on the current connector.

**Comment by Congyuwang** [comment]
attempted fix using feature: Fix #174
